src/frontend/src/components/RecordSearch/Record/TypeEligibility.tsx

```diff
@@ -24,7 +27,7 @@ export default class RecordType extends React.Component<Props> {
       <div className="relative mb3 connect connect-type">
         <i
           aria-hidden="true"
-          className="absolute fas fa-question-circle purple bg-white z-1"
+          className="absolute fas fa-question-circle purple bg-white outline-2-white z-1"
```

Comment:
@hmarcks I was having trouble figuring out where this style update came from, and not sure if I put it back in there by accident. Do we still want this change to go in? Same thing for Time Eligibility.

Reply:
Yes please! Confusing because I snuck it in this branch but not related to the issue